ecpy/utils/widgets/qt_list_str_widget.py

```diff
@@ -99,9 +99,21 @@ def on_selection(self):
                        for index in widget.selectedIndexes()]
             if indexes:
                 if self.multiselect:
+                    # HINT force an access so that the new change (which is
+                    # swallowed by enaml) so that the notifications are
+                    # properly propagated
+                    self.selected_indexes
+                    self.selected_items
+
                     self.selected_indexes = indexes
                     self.selected_items = [self.items[i] for i in indexes]
                 else:
+                    # HINT force an access so that the new change (which is
+                    # swallowed by enaml) so that the notifications are
+                    # properly propagated
+                    self.selected_index
+                    self.selected_item
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://www.codacy.com/app/Ecpy/ecpy/file/2642575397/issues/source?bid=3033427&fileBranchId=3321358#l115)